site/docs/v1/tech/identity-providers/_idp-managed-domains-troubleshooting.adoc

```diff
@@ -0,0 +1,25 @@
+A common pattern is to set up {idp_type_name} as the login method for the FusionAuth application. This lets you keep all your users in the {idp_type_name} system, and have FusionAuth delegate auth decisions to it. However, if your Identity Provider isn't configured correctly, you can end up in a dead end, unable to log in. This might happen if you:
+
+* Set up {idp_type_name} as the authentication method for your FusionAuth admin application.
+* Set the [field]#Managed domains# to your domain. For example, `example.com`.
+* Don't have any users with a domain that is different than `example.com`.
+* Accidentally misconfigure {idp_type_name} and break authentication.
+* Log out of the FusionAuth admin application.
+
+You won't be able to log in to the FusionAuth admin application to correct the misconfiguration. You've locked down the FusionAuth admin application so that only valid logins can access it, but because of the misconfiguration, there is no account with a valid {idp_type_name} login.
+
+image::troubleshooting/unable-to-login-managed-domains.png[Unable to login with a managed domain and misconfigured Identity Provider.,width=1200,role=bottom-cropped]
+
+This is a problem, but not an insurmountable one. Your options depend on when you discover the issue. If you are beginning your {idp_type_name} configuration, you can avoid this scenario. Follow these steps:
+
+* Don't set any value of [field]#Managed domains# before you have tested the {idp_type_name} configuration. 
+* Test authentication in a different or incognito browser window, ensuring that an admin user account is always accessible to modify configuration.
+* Add an admin user account with a domain not in the [field]#Managed domains# setting. Ensure this user is registered with the FusionAuth admin application and has the `admin` role.
+
+If you are currently locked out of your FusionAuth application, you have fewer options:
+
+* If you have an API key with appropriate privileges you can delete the misconfigured identity provider. You can list all identity providers and then delete it. See the link:/docs/v1/tech/apis/identity-providers/[relevant identity provider API for more]. This will then enable normal login.
```

Comment:
This is one option for sure. I suppose we want to balance the difficulty here, but with an API key you could also:
- Create a FusionAuth admin user with an email domain not managed by the IdP so you can login
- Disable the IdP configuration
- Disable the IdP configuration just for the FusionAuth admin applications, or all applications
- Remove the domain configuration from the IdP
- Correct the IdP configuration, if this can be done w/out testing (maybe they fat fingered something simple)

These options are not destructive so they configuration can be re-used or modified after they get back into the UI.

Reply:
Great. I picked the simplest solution (add a new admin user) and updated the doc.